<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
background-color: rgb(36, 39, 45);
color:white
}</string>
    </property>
    <property name="text">
     <string>2+3-4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="diverce">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_minus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(90, 92, 97);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(111, 115, 121);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Brace_open">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Brace_close">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xprint">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(216, 212, 241);
	color:rgb(55, 55, 55);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(169, 166, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_equation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>5</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="graph">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>50</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(114, 86, 124);
	border-width: 1px;
    border-style: outset;
}
QPushButton:pressed {
    background-color: rgb(96, 72, 105);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
